### PR TITLE
Minor updates for AMI and ICSI

### PR DESCRIPTION
**Pull request by desh2608**
This PR contains some minor updates for the AMI and ICSI recipes that were made during mini SCALE.
* New Kaldi-style text normalization option
* Handling some AMI sessions where headset-mics are stereo
* Minor bug fix for ICSI channel names

### CODE REVIEW

File: lhotse/recipes/ami.py [hunk 1]
```diff
@@ -265,7 +265,7 @@ class AmiSegmentAnnotation(NamedTuple):
 
 
 def parse_ami_annotations(
-    annotations_dir: Pathlike, normalize_text: bool = True
+    annotations_dir: Pathlike, normalize: str = "upper"
```
Comment (pzelasko) [inline lhotse/recipes/ami.py:268]: why not make "kaldi" default here as well for consistency?

File: lhotse/recipes/ami.py [hunk 2]
```diff
@@ -343,19 +341,44 @@ def _parse_href(href, wid_to_word):
...
+                            start_time=start_time,
+                            end_time=end_time,
+                        )
                     )
-                )
 
     return annotations
 
 
+def normalize_text(text: str, normalize: str = "upper") -> str:
```
Comment (pzelasko) [inline lhotse/recipes/ami.py:359]: ... and here

File: lhotse/recipes/icsi.py [hunk 3]
```diff
@@ -126,9 +127,9 @@
 }
 
 MIC_TO_CHANNELS = {
-    "ihm": [1, 2, 3, 4, 5, 6, 8, 9], # we include 6 since it is used as back-off from some speakers for which no headset-mic exists
-    "sdm": [6],
-    "mdm": ["E", "F", 6, 7],
+    "ihm": ["1", "2", "3", "4", "5", "6", "8", "9"], # we include 6 since it is used as back-off from some speakers for which no headset-mic exists
```
Comment (pzelasko) [inline lhotse/recipes/icsi.py:130]: what's the reason these are represented as strings now?

**Review by pzelasko — COMMENTED** [review]
Thanks! I left a bunch of comments

File: lhotse/recipes/icsi.py (hunk 3 above)
Comment (LasseWolter) [reply lhotse/recipes/icsi.py:130]: If not converted to string this line won't work: 

`audio_paths = audio_dir.rglob(f"chan[{channels}].sph")`
https://github.com/lhotse-speech/lhotse/blob/master/lhotse/recipes/icsi.py#L505

**Comment by LasseWolter** [comment]
@desh2608 because you worked with the ICSI recipe, you might know more about  #551

**Comment by pzelasko** [comment]
LGTM

File: lhotse/recipes/ami.py (hunk 1 above)
Comment (desh2608) [reply lhotse/recipes/ami.py:268]: Makes sense, I'll do it next time I raise a PR.

File: lhotse/recipes/icsi.py (hunk 3 above)
Comment (desh2608) [reply lhotse/recipes/icsi.py:130]: @LasseWolter thanks, that's exactly the reason. Sorry I was traveling last couple of days so couldn't reply to these comments.

**Pull request merged**